File: codeforces/785A - Anton and Polyhedrons/a.py
def main():
    n = int(input())
    hm = {'Tetrahedron':4,'Cube':6,'Octahedron':8,'Dodecahedron':12,'Icosahedron':20}
    ans = 0
    for _ in range(n):
        curr = str(input())
        ans += hm[curr]
    print(ans)
                              
                                    
if __name__ == "__main__":
    main()

File: codeforces/785A - Anton and Polyhedrons/a.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int n;
    cin >> n;
    int ans = 0;
    unordered_map<string, int> hm = {{"Tetrahedron",4},
                                     {"Cube",6},  
                                     {"Octahedron",8},
                                     {"Dodecahedron",12},
                                     {"Icosahedron",20}
                                    };
    for(int i = 0; i < n; i++) {
        string curr;
        cin >> curr;
        ans += hm[curr];
    }
    cout << ans;
    return 0;
}